install-pcf/aws/tasks/prepare-aws/task.sh

```diff
@@ -8,14 +8,14 @@ OPSMAN_ALLOW_SSH=0
 OPSMAN_ALLOW_SSH_CIDR_LIST='["0.0.0.0/32"]'
 if [[ -n "${OPSMAN_ALLOW_SSH_CIDR_RANGES// }" ]]; then
   OPSMAN_ALLOW_SSH=1
-  OPSMAN_ALLOW_SSH_CIDR_LIST='["'${OPSMAN_ALLOW_SSH_CIDR_RANGES/\,/\"\,\"}'"]'
+  OPSMAN_ALLOW_SSH_CIDR_LIST='["'${OPSMAN_ALLOW_SSH_CIDR_RANGES//\,/\"\,\"}'"]'
```

Comment:
Actually, I'm not sure of the point of the line anyway. Why not just pass in a quoted string in the params line?